Formula/fzfnpm.rb

```diff
@@ -0,0 +1,19 @@
+class Fzfnpm < Formula
+  desc "Fuzzy finder npm script runner"
+  homepage "https://github.com/FrancisVega/fzfnpm"
+  url "https://github.com/FrancisVega/fzfnpm/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "883cf5df0a76abbd22726609d8f516134e23350bfcb7fee5f2d29b1a5a0dc147"
+
+  license "MIT"
+
+  depends_on "bash"
+  depends_on "fzf"
+
+  def install
+    bin.install "bin/fzfnpm"
+  end
+
+  test do
+    assert_predicate Pathname.new("#{bin}/fzfnpm"), :executable?
```

Comment:
Can you propose a more substantial test?

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Yes, I will do my best to do it thanks!

Reply:
I add a test to check if a package.json file exists. If not , the script exit with a message and exit code 1